src/utils/textToSpeech.js

```diff
@@ -0,0 +1,19 @@
+import axios from 'axios';
+
+
+const textToSpeech = async (text) => {
+    const content = {text: text};
+    try{
+        const response = await axios.post('http://localhost:3000/texts/text-to-speech', content)
+        console.log(response.data)
```

Comment:
can we log a success message like "Text to speech success" instead of printing response.data to avoid printing a very long base64 string when we start converting longer texts